src/solvers/dgsem_p4est/containers_2d.jl

```diff
@@ -6,7 +6,7 @@
 
 
 # Initialize data structures in element container
-function init_elements!(elements, mesh::P4estMesh{2}, basis::LobattoLegendreBasis)
+function init_elements!(elements, mesh::Union{P4estMesh{2},T8codeMesh{2}}, basis::LobattoLegendreBasis)
```

Comment:
```suggestion
function init_elements!(elements, mesh::Union{P4estMesh{2}, T8codeMesh{2}}, basis::LobattoLegendreBasis)
```
Please search for this pattern and update it with a space

Reply:
Done.